src/renderer/editor/selection.ts

```diff
@@ -26,3 +27,30 @@ export const selectAllWords: () => void = () => {
     })
   );
 };
+
+export const selectSentence: () => void = () => {
+  const { currentProject } = store.getState();
+
+  if (currentProject === null || currentProject?.transcription === null) {
+    return;
+  }
+  let { startIndex, endIndex } = store.getState().selection.self;
+  endIndex -= 1; // Account for the fact that the selection is exclusive of the end index
```

Comment:
nice copilot comment 😉

Reply:
Hahaha I was pretty impressed it came up with this one